src/robusta/core/sinks/opsgenie/opsgenie_sink.py

```diff
@@ -0,0 +1,142 @@
+import logging
+import re
+
+import opsgenie_sdk
+
+from typing import List
+from tabulate import tabulate
+
+from .opsgenie_sink_params import OpsGenieSinkConfigWrapper
+from ...reporting.blocks import (
+    KubernetesDiffBlock,
+    JsonBlock,
+    MarkdownBlock,
+    HeaderBlock,
+    ListBlock,
+    TableBlock,
+    DividerBlock,
+)
+from ...reporting.base import (
+    Finding,
+    Enrichment,
+    FindingSeverity,
+)
+from ..sink_base import SinkBase
+
+
+PRIORITY_MAP = {
+    FindingSeverity.INFO: "P5",
+    FindingSeverity.LOW: "P4",
+    FindingSeverity.MEDIUM: "P3",
+    FindingSeverity.HIGH: "P1"
```

Comment:
Can you add a comment on why we're skipping P2 here?

Reply:
Robusta has only 4 severities, and OpsGenie has 5. We had to omit 1 priority
I'll add a comment on that